install/Knative-custom-install.md

```diff
@@ -155,23 +155,23 @@ The following Knative installation packages are available:
 
 - **Multiple Component Bundle**:
 
-  - https://github.com/knative/serving/releases/download/v0.2.3/release.yaml
-  - https://github.com/knative/serving/releases/download/v0.2.3/release-no-mon.yaml
-  - https://github.com/knative/serving/releases/download/v0.2.3/release-lite.yaml
+  - https://github.com/knative/serving/releases/download/v0.3.0/release.yaml
```

Comment:
Please revert this file. A lot of this info needs to be entirely rewritten for 0.3.0.

Reply:
I rebased to remove the commit that changed this file. PTAL!

Reply:
thanks! looks good.